easylist/easylist_specific_hide.txt

```diff
@@ -4176,7 +4176,7 @@ tetris.com##a[onclick*="open"]
 himovies.to,home-barista.com,rarpc.co,shtfplan.com,warecracks.com,washingtontimes.com##a[onclick]
 onetransistor.eu##a[onclick][target="_blank"]
 earlygame.com,i711.com##a[rel*="sponsored"]
-amishamerica.com##a[rel="nofollow"]
+amishamerica.com,flixtor.video##a[rel="nofollow"]
```

Comment:
This is too broad, although doesn't cause false positive on the sites. I'll change after merge.